packages/dds/tree/src/simple-tree/api/conciseTree.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { IFluidHandle } from "@fluidframework/core-interfaces";
+
+import type { ITreeCursor } from "../../core/index.js";
+import type { TreeLeafValue, ImplicitAllowedTypes } from "../schemaTypes.js";
+import type { TreeNodeSchema } from "../core/index.js";
+import { customFromCursorInner, type EncodeOptions } from "./customTree.js";
+import { walkFieldSchema } from "../walkFieldSchema.js";
+
+/**
+ * Concise encoding of a {@link TreeNode} or {@link TreeLeafValue}.
+ * @remarks
+ * This is concise meaning that explicit type information is omitted.
+ * If the schema is compatible with {@link ITreeConfigurationOptions.preventAmbiguity},
+ * types will be lossless and compatible with {@link TreeBeta.create} (unless the options are used to customize it).
+ *
+ * Every {@link TreeNode} is an array or object.
+ * Any IFluidHandle values have been replaced by `THandle`.
+ * @privateRemarks
+ * This can store all possible simple trees,
+ * but it can not store all possible trees representable by our internal representations like FlexTree and JsonableTree.
+ * @beta
```

Comment:
Most of the new items in this PR are tagged as `@beta`, but nothing new was added to the package exports. Probably best to remove the tags until we're ready to export them?

Reply:
If left them to avoid conflicts with the branch these were extracted from (which does export these). If you are ok leaving these currently no-op tags here until the next changes land, I'd like to do so. I can remove them if you think I should though.

Reply:
Looks like we have other docs changes anyway, so I'll just remove them